src/reducers/auth.ts

```diff
@@ -0,0 +1,25 @@
+import * as Constants from 'constants/auth'
+
+/* eslint-disable  @typescript-eslint/no-explicit-any */
+type ActionType = {
+  type: string
+  payload: any
+}
+
+const AuthReducer = (state: Record<string, unknown>, action: ActionType): Record<string, unknown> => {
```

Comment:
`Record<string, unknown>` usually means `any object` but I think the `AuthReducer` should have a fixed return type. If the `state` object is defined then `AuthReducer` should have a specific return type as it is returning a state.

Reply:
Fixed in e187091b02cc6415f847925c96333565b9580b51